crates/forge/src/fuzzer.rs

```diff
@@ -1,11 +1,35 @@
 use cairo_felt::Felt252;
 use num_bigint::{BigUint, RandBigInt};
+use num_traits::One;
 use rand::rngs::StdRng;
-use rand::{thread_rng, RngCore, SeedableRng};
+use rand::{thread_rng, Rng, RngCore, SeedableRng};
+use std::default::Default;
+
+#[derive(Default)]
+#[allow(clippy::module_name_repetitions)]
+pub struct FuzzerRunParams {
+    /// Inclusive value
+    low: BigUint,
+    /// Exclusive value
+    high: BigUint,
```

Comment:
We will always generate felts, those have to be configurable?

Reply:
I guess we can just do another PR with enabling other types of args. Then we can think about how to solve this - we can have args of types `u8`, `u16` and `felt252` in one function and then simple generic is not enough

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/693